container-guide/concepts/containers-build-app-images.xml

```diff
@@ -414,7 +414,7 @@
       <listitem>
         <para>
           Point a browser to
-          <link xlink:href="http://localhost:80/test.html">http://localhost:80/test.html</link>
```

Comment:
@dmpop : I assume here the replacement does not work, please check!

Reply:
Correct: it's localhost, so http is the only option.